EXILED/Exiled.Events/Patches/Fixes/VoiceChatTpsFix.cs

```diff
@@ -0,0 +1,48 @@
+// -----------------------------------------------------------------------
+// <copyright file="VoiceChatTpsFix.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Fixes
+{
+    using System.Collections.Generic;
+    using System.Configuration;
+    using System.Reflection.Emit;
+
+    using API.Features.Pools;
+
+    using HarmonyLib;
+
+    using VoiceChat.Networking;
+
+    using static HarmonyLib.AccessTools;
+
+    /// <summary>
+    /// Fixes <see cref="VoiceTransceiver.ServerReceiveMessage(Mirror.NetworkConnection, VoiceChat.Networking.VoiceMessage)"/> method.
+    /// </summary>
+    [HarmonyPatch(typeof(VoiceTransceiver), nameof(VoiceTransceiver.ServerReceiveMessage))]
+    internal static class VoiceChatTpsFix
+    {
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator generator)
+        {
+            List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Pool.Get(instructions);
+            int offset = 0;
+            int index = newInstructions.FindIndex(instruction => instruction.opcode == OpCodes.Newarr) + offset;
+            newInstructions.InsertRange(index, new CodeInstruction[]
+            {
+                // popped 24000
+                new CodeInstruction(OpCodes.Pop),
+
+                // loadded 480
+                new CodeInstruction(OpCodes.Ldc_I4, 480),
+            });
```

Comment:
That can be shortcuted to that
```suggestion
            int offset = -1;
            int index = newInstructions.FindIndex(instruction => instruction.opcode == OpCodes.Newarr) + offset;
            newInstructions[index].operand = 480;
```
not to pop already loaded `24000`, just changing loading number😊

Reply:
oh, thy, i am new in transpilers, didnt know that. Anyway this code dont work as planned, still testing it.